Makefile

```diff
@@ -0,0 +1,24 @@
+TOP_LEVEL=$(shell git rev-parse --show-toplevel)
+TMPDIR := $(shell mktemp -d)
+TOOLSDIR := $(shell pwd)/hack/tools
+SHELL = /bin/bash
+BATS := $(TOOLSDIR)/bin/bats
+BATS_FLAGS = --verbose-run --show-output-of-passing-tests --print-output-on-failure
+
+.PHONY: all
+all: conformance
+
+$(BATS):
+	pushd $(TMPDIR); \
+	git clone https://github.com/bats-core/bats-core.git; \
+	cd bats-core; ./install.sh $(TOOLSDIR); cd ..; \
```

Comment:
`cd ..` is not needed since you call `popd` right after that.

Reply:
Fixed.